Selecting NANO sockets breaks build

net/net.a(socket.o):function sock_awaitconn: error: undefined reference to 'clr_irq'
net/net.a(socket.o):function sock_awaitconn: error: undefined reference to 'set_irq'

--- a/elks/net/nano/af_nano.c
+++ b/elks/net/nano/af_nano.c
@@ -15,6 +15,8 @@
 #include <linuxmt/mm.h>
 #include <linuxmt/stat.h>
 #include <linuxmt/fcntl.h>
+
+#include <arch/irq.h>
 #include "af_nano.h"
 
 #ifdef CONFIG_NANO

--- a/elks/net/socket.c
+++ b/elks/net/socket.c
@@ -26,6 +26,7 @@
 #include <linuxmt/debug.h>
 
 #include <arch/segment.h>
+#include <arch/irq.h>
 
 #define socki_lookup(inode)	(&inode->u.socket_i)
 

--- a/elks/net/unix/af_unix.c
+++ b/elks/net/unix/af_unix.c
@@ -18,6 +18,7 @@
 #include <linuxmt/debug.h>
 
 #include <arch/segment.h>
+#include <arch/irq.h>
 #include "af_unix.h"
 
 #ifdef CONFIG_UNIX

